python/vyos/ifconfig/interface.py

```diff
@@ -739,6 +781,11 @@ def set_ipv4_source_validation(self, value):
 
         As per RFC3074.
         """
+
+        # Don't allow for netns yet
+        if self.netns:
```

Comment:
instead of silently hiding this we should use common verify() rules and inform the user

Reply:
But this method will be per any interface change anyway. Even if we change the 'description,' isn't it?

Reply:
You are right - that code-path could be triggered by the interface defaults